package.json

```diff
@@ -189,24 +186,13 @@
     "prephenomic-theme-base-start": "npm -s run transpile && babel-node scripts/phenomic-theme-base.js",
     "phenomic-theme-base-start": "cd themes/phenomic-theme-base && npm start",
     "preall-tests": "npm run lint",
-    "all-tests": "cross-env TESTING=1 npm run tests && npm run test-phenomic-theme-base && npm run docs",
-    "test": "npm run all-tests",
-    "test-with-coverage": "nyc --check-coverage --lines 70 npm run all-tests",
-    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "all-tests": "cross-env TESTING=1 jest --coverage",
+    "postall-tests": "npm run test-phenomic-theme-base && npm run docs",
+    "test": "cross-env TESTING=1 jest",
```

Comment:
There is 1 problem with using NODE_ENV=test. The configurator changes NODE_ENV when we use `--production` flag. And it kills jest